infrastructure-aws-production.yml

```diff
@@ -1,14 +1,6 @@
-resource_pools:
-- name: minion
-  cloud_properties:
-    instance_type: r4.xlarge
-- name: etcd
-  cloud_properties:
-    instance_type: t2.medium
-
 jobs:
 - name: minion
   instances: 7
   networks:
-  - name: default
-    static_ips: (( static_ips(9, 10, 11, 12, 13, 14, 15) ))
+  - name: minion
```

Comment:
This allocates all static ips to the kubernetes minions. Unless the other instance groups in this deployment (etcd, consul, master) are getting moved to dynamic ip addresses, we should keep the addresses the same, like this:
```yaml
static_ips:
- (( grab terraform_outputs.kubernetes_static_ips.[9]
- (( grab terraform_outputs.kubernetes_static_ips.[10]
...
```